### PR TITLE
Changelog for qcodes 0.21.0 (Unreleased)

### PR DESCRIPTION
**Pull request by FarBo**
I am opening the early changelog for Qcodes 0.21.0 with some of the changes that we have for now. Feel free to update it continuously until the next release.

@jenshnielsen 
@astafan8 
@WilliamHPNielsen 
@Akshita07 
@trevormorgan

### CODE REVIEW

File: docs/changes/0.21.0.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+----
+New:
+----
+-
+
+---------
+Improved:
+---------
+- The old loop and plotting methods are removed from driver notebooks (#2477)
```
Comment (jenshnielsen) [inline docs/changes/0.21.0.rst:20]: Make it clear that this is alazar driver

**Comment by jenshnielsen** [comment]
for this to build cleanly you will need to link this into the index file in the same folder

File: docs/changes/0.21.0.rst [hunk 2]
```diff
@@ -0,0 +1,35 @@
+Changelog for QCoDeS 0.21.0
```
Comment (jenshnielsen) [inline docs/changes/0.21.0.rst:1]: Please add a note that this is unreleased. `Changelog for QCoDeS 0.21.0 (unreleased)`

**Review by jenshnielsen — APPROVED** [review]

**Comment by jenshnielsen** [comment]
@FarBo can you fix the rst formatting.  The underline needs to be the same length as the text

**Review by trevormorgan — APPROVED** [review]

File: docs/changes/0.21.0.rst [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- The default behavior of do_plot in doNd functions changed to False.
+  The default value can be configured in the qcodesrc.json config file (#2478)
+- plotting: Don't scale empty units (#2488)
+
+----
+New:
+----
+
+- Example notebook for using Plottr with Qcodes measurements for live plotting (#2494)
+- Add kwarg for experiment id to doNd functions (With tests)  (#2526)
```
Comment (astafan8) [inline docs/changes/0.21.0.rst:19]: it's not id, it's the object, right?
```suggestion
- Add kwarg for passing experiment to doNd functions (With tests)  (#2526)
```

File: docs/changes/0.21.0.rst [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- Add PycQED to related projects (#2446)
+- Updated HP8753D notebook with doNd and its driver timeout (#2520)
+
+-----------------
+Improved drivers:
+-----------------
+
+- Keysight 34465A: Add missing parameters to pyvisa sim yaml file (#2452, #2493)
+- Driver SR86x: Add complex voltage (#2458)
+- Fix parameter name in ats9870: EXTERNAL_CLOCK_10MHz_REF (#2492)
```
Comment (astafan8) [inline docs/changes/0.21.0.rst:35]:
```suggestion
- Fix parameter name in Alzar ATS9870: EXTERNAL_CLOCK_10MHz_REF (#2492)
```

File: docs/changes/0.21.0.rst [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+- Keysight B1500: ensure making snapshot of the pyvisa-sim instrument does not raise warnings (#2532)
+
+---------------
+Under the hood:
+---------------
+
+- Verify that requirements are compatible with Spyder (#2443)
+- Require use of Any in all generic types (#2450, #2453, #2454, #2457, #2463, #2468, #2469)
+- Minor typing improvements (#2456, #2475)
+- refactor logic to find scale and prefix into public function (#2480)
```
Comment (astafan8) [inline docs/changes/0.21.0.rst:47]:
```suggestion
- refactor logic to find scale and prefix for units into a public function (#2480)
```

**Review by astafan8 — APPROVED** [review]

File: docs/changes/0.21.0.rst [hunk 6]
```diff
@@ -0,0 +1,48 @@
+Changelog for QCoDeS 0.21.0
+===========================
+
+The December 2020 release of QCoDeS.
```
Comment (jenshnielsen) [inline docs/changes/0.21.0.rst:4]: https://github.com/QCoDeS/Qcodes/pull/2511 seems to be missing

**Pull request merged**